### PR TITLE
Add agile methods article to assets

### PR DESCRIPTION
**Pull request by mleimeister**
Add the English translation of Timo Daum's "Agile Methods" article to assets for the upcoming learning group.

### CODE REVIEW

**Comment by chobeat** [comment]
@mleimeister are you familiar with Jekyll? do you feel confident including the article in the website and link it somewhere?

@shushugah we don't really have a section for articles, shall we add one?

File: assets/Timo_Daum-Agile_Methods_(English).md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+In all situations of life we are now called upon to measure ourselves, to take action, to take risks, to become our own CEO. As individuals, we are called upon to look at ourselves as entrepreneurs. Our own self becomes human capital and must consequently be optimised in terms of business management: Staying on a diet, going to the gym - no aspect of life is left out when optimizing. Quantified Self is the realization of Taylor's program of a scientific management of the resource of living labor, this time shifted into the individual.
```
Comment (chobeat) [inline assets/Timo_Daum-Agile_Methods_(English).md]: > As individuals, we are called upon to look at ourselves as entrepreneurs.

I would replace "called upon" with "asked". Called upon is way too strong given the context.

Comment (chobeat) [reply assets/Timo_Daum-Agile_Methods_(English).md]: > Staying

The S should be lowercase

**Comment by mleimeister** [comment]
@chobeat I'm not familiar with Jekyll unfortunately. I tried building the site locally to test adding a link to the events page but get the following error both with docker and bundler:

```
jekyll_1  | jekyll 3.8.5 | Error:  Could not locate the included file 'generated/calendar.js' in any of ["/srv/jekyll/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source.
```

Am I missing some dependencies? If it is working for you, could you add a link after merging?

**Comment by chobeat** [comment]
I think you have to run `npm install`.

**Comment by shushugah** [comment]
Thank you for translating this Mathius! I will make some changes tonight, so we can make this go live!

Hey sorry about the NPM dependencies, I'll update the README instructions. It's for this calendar here: https://techworkersberlin.com/calendar/

I would prefer we create a "post collection", so that we can have a feed of posts, that are similar to events, but include extra info like authors etc.. and also show up in SEO. 
This link doesn't preview/generate html, because assets skip the compiling
https://deploy-preview-66--techworkersberlin.netlify.com/assets/Timo_Daum-Agile_Methods_(English).md

**Comment by shushugah** [comment]
Have a look at the code here, and let me know if it makes sense! https://github.com/techworkersco/twc-site-berlin/pull/68

### index
https://deploy-preview-68--techworkersberlin.netlify.com/

### individual article
https://deploy-preview-68--techworkersberlin.netlify.com/news/2020-04-01-timo_daum-agile_methods_english/

### articles

https://deploy-preview-68--techworkersberlin.netlify.com/news

**Comment by shushugah** [comment]
Closed because #68 is merged

**Pull request closed without merging**